TeamCode/src/main/java/org/firstinspires/ftc/teamcode/team_classes/Lift.java

```diff
@@ -0,0 +1,45 @@
+package org.firstinspires.ftc.teamcode.team_classes;
+
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+import org.firstinspires.ftc.robotcore.external.navigation.Position;
+import org.firstinspires.ftc.teamcode.general_classes.Position2D;
+import org.firstinspires.ftc.teamcode.general_classes.Position2DAngle;
+
+public class Lift extends DcMotorGroup {
+    //Constant Properties
+    private static final double COUNTS_PER_MOTOR_REV = 753.2;    // SKU: 5202-0002-0027
+    private static final double DRIVE_GEAR_REDUCTION = 1.0;     // This is < 1.0 if geared UP
+    private static final double WHEEL_DIAMETER_MM = 100;       // goBilda metric wheels are 100mm
```

Comment:
This value will have to be changed depending on the size of gearing/pulleying we use on the bot.